cmd/fyne/internal/commands/build.go

```diff
@@ -7,16 +7,72 @@ import (
 	"strings"
 
 	version "github.com/mcuadros/go-version"
+	"github.com/urfave/cli/v2"
 )
 
-type builder struct {
+// Builder generate the executables.
+type Builder struct {
 	os, srcdir, target string
 	release            bool
 	tags               []string
+	tagsToParse        string
 
 	runner runner
 }
 
+// Build returns the cli command for building fyne applications
+func Build() *cli.Command {
+	b := &Builder{}
+
+	return &cli.Command{
+		Name:        "build",
+		Usage:       "Build an application.",
+		Description: "You may specify the --executable to build, but --target will define the OS it is build for indiscriminate of the extention of the executable.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "target",
+				Aliases:     []string{"os"},
+				Usage:       "The mobile platform to target (android, android/arm, android/arm64, android/amd64, android/386, ios, iossimulator).",
+				Destination: &b.os,
+			},
+			&cli.StringFlag{
+				Name:        "sourceDir",
+				Aliases:     []string{"src"},
+				Usage:       "The directory to package, if executable is not set.",
+				Destination: &b.srcdir,
+			},
+			&cli.StringFlag{
+				Name:        "tags",
+				Usage:       "A comma-separated list of build tags.",
+				Destination: &b.tagsToParse,
+			},
+			&cli.BoolFlag{
+				Name:        "release",
+				Usage:       "Enable installation in release mode (disable debug etc).",
+				Destination: &b.release,
+			},
+			&cli.StringFlag{
+				Name:        "executable",
```

Comment:
This seems not needed - as the exe is the result of build, so why would we pass in an alternative binary?

Reply:
Maybe it should match _go build_ command line and be _-o_ instead?

Reply:
That sounds like a splendid idea.

Reply:
I wonder, is it better to keep consistency with other `fyne` commands, or stay close to the regular `go build`? I think the latter would make it easier to use for new users at the expense of us old chaps ;)

Reply:
This looks confusing to me - the executable is an input parameter for all the other commands. file names are set according to the `name` parameter.
If we are to change this it should be for all commands, not just this one.